jvb/src/main/kotlin/org/jitsi/videobridge/relay/RelayMessageTransport.kt

```diff
@@ -195,6 +195,8 @@ class RelayMessageTransport(
 
         ep.setVideoType(message.videoType)
 
+        relay.conference.sendMessageFromRelay(message, relay.meshId)
```

Comment:
This broadcasts the received video type to all local endpoints. Is this the intention?

Reply:
No, will fix.